Makefile

```diff
@@ -0,0 +1,48 @@
+###############################################################################
```

Comment:
As previously discussed, a Makefile is a very poor approach to dependency installation when `pip` is sufficient for the job, is the standard Python tool on all platforms, and is already configured in this repository.